tech/languages/haskell/leksah.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Leksah
+subsection: haskell
+order: 4
+---
+
+# Leksah
+
+[Leksah](http://leksah.org/) is a Haskell IDE written in Haskell using Gtk. It offers Workspaces for complex project with multiple 
```

Comment:
I think "Workspaces" should be either bolded or lowercased. Can you please rephrase it a bit?